src/Mvc/test/Mvc.FunctionalTests/FileResultTests.cs

```diff
@@ -288,7 +288,8 @@ public async Task FileFromDisk_ReturnsFileWithFileName_IfRangeHeaderInvalid_Rang
         Assert.Equal("attachment; filename=downloadName.txt; filename*=UTF-8''downloadName.txt", contentDisposition);
     }
 
-    [Fact]
+    [ConditionalFact]
+    [OSSkipCondition(OperatingSystems.Windows)]
```

Comment:
Do you think it's worth a comment explaining why it's skipped?

Reply:
```suggestion
    [OSSkipCondition(OperatingSystems.Windows)] // Creating symlinks requires special permissions on Windows
```